docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -168,17 +168,22 @@ There are [client libraries](/docs/reference/client-libraries/) for accessing th
 
 ### Accessing the API from a Pod
 
-When accessing the API from a pod, locating and authenticating
+When accessing the API from a Pod, locating and authenticating
 to the API server are somewhat different.
 
-The recommended way to locate the apiserver within the pod is with
-the `kubernetes` DNS name, which resolves to a Service IP which in turn
-will be routed to an apiserver.
+The easiest way to use the Kubernetes API from a Pod is to use
+one of the official [client libraries](/docs/reference/client-libraries/). These
+libraries can automatically discover the apiserver and authenticate automatically.
 
```

Comment:
I don't think you need the second "automatically".
... can automatically discover the API server and authenticate.

Reply:
done